README.md

```diff
@@ -409,9 +419,9 @@ istio-egressgateway-75c5457c56-htpvz   1/1     Running   0          33m   10.244
 
 <img src=passthrough-egress.png>
 
-This setup is useful when you want to enforce policies and monitor egress traffic while allowing the destination to manage its own TLS.
+This setup is useful when you want to enforce policies and monitor egress traffic while allowing the destination to manage its own TLS. Traffic leaving via the egress gateway will perform SNI passthrough instead of TLS terminatation for requests. 
```

Comment:
```suggestion
This setup is useful when you want to enforce policies and monitor egress traffic while allowing the destination to manage its own TLS. Traffic leaving via the egress gateway will perform SNI passthrough instead of TLS termination for requests. 
```